stratum/hal/bin/tdi/dpdk/dpdk_main.cc

```diff
@@ -30,10 +30,11 @@
 #include "stratum/lib/security/auth_policy_checker.h"
 
 #define CONFIG_PREFIX "/usr/share/stratum/dpdk/"
```

Comment:
I'd suggest renaming these to `DEFAULT_CONFIG_PREFIX` and `DEFAULT_LOG_DIR`.

Reply:
Done

Reply:
Incidentally, my observations on the comments are equally applicable to the changes I made. It didn't occur to me until I looked the code anew (and in light of the increasing amount of customization) that more and better documentation would be appropriate. ("Do as I should have done, not as I did.")